pkg/api/protobuf/go/minder/v1/validators.go

```diff
@@ -286,6 +293,32 @@ func (ing *RuleType_Definition_Ingest) Validate() error {
 	return nil
 }
 
+// Validate validates a rule type definition alert
+func (alert *RuleType_Definition_Alert) Validate() error {
+	if alert == nil {
```

Comment:
Wait, alert should be nullable... shouldn't it?

Reply:
Nevermind, noticed the check above

Reply:
Yes, but I figured if it gets to the point of being validated then it shouldn't be null, as in the caller should decide on the nullability. I can also change this to `return nil` if the alert is `nil`, I don't feel strongly.

Reply:
I think changing it to return `nil` is fine, as alert configuration is optional after all.

Reply:
Updated